source/config.civet

```diff
@@ -0,0 +1,72 @@
+import path from "path";
```

Comment:
Not a high priority, but now that we're using Civet instead of TS, we could make this code closer to the CoffeeScript version you're replacing here (but with type annotations). Could be saved for the future though.